crates/blockifier/src/execution/entry_point.rs

```diff
@@ -211,14 +223,18 @@ pub struct CallExecution {
 }
 
 #[derive(Debug, Default, Eq, PartialEq)]
+#[cfg_attr(feature = "parity-scale-codec", derive(Encode, Decode))]
 pub struct CallInfo {
     pub call: CallEntryPoint,
     pub execution: CallExecution,
+    #[cfg_attr(feature = "parity-scale-codec", codec(skip))]
     pub vm_resources: VmExecutionResources,
     pub inner_calls: Vec<CallInfo>,
 
     // Additional information gathered during execution.
+    #[cfg_attr(feature = "parity-scale-codec", codec(skip))]
     pub storage_read_values: Vec<StarkFelt>,
+    #[cfg_attr(feature = "parity-scale-codec", codec(skip))]
```

Comment:
Is there no way for you to have codec implemented on those types?
Are they not part of our forks?